pkg/executor/reaper/reaper.go

```diff
@@ -71,108 +71,116 @@ func CleanupKubeObject(ctx context.Context, logger *zap.Logger, kubeClient kuber
 
 // CleanupDeployments deletes deployment(s) for a given instanceID
```

Comment:
Please revert the changes in the file for now.

Reply:
done